backend/hyper/entity.go

```diff
@@ -324,7 +325,9 @@ func (bs *Storage) ForEachChange(ctx context.Context, eid EntityID, fn func(c ci
 }
 
 // LoadEntity from the database. If not found returns nil result and nil error.
-func (bs *Storage) LoadEntity(ctx context.Context, eid EntityID) (e *Entity, err error) {
+// If trustedOnly true, then it will returnl the lastest version changed by a trusted peer.
+// It will return the latest version available otherwhise, regardles on who made the change.
+func (bs *Storage) LoadEntity(ctx context.Context, eid EntityID, trustedOnly bool) (e *Entity, err error) {
```

Comment:
I'd like to avoid adding boolean arguments to functions, because Go doesn't have named arguments, and it's hard to understand what the boolean means without looking up the function signature.

I think a better way to implement this would be by creating a new function to get trusted-only heads. We now have a function to get all public heads, which we then pass to load entity function. I think we should make a function to get trusted heads, and use the same function for loading an entity without any modifications.

So the process for loading all entities would be the same - find the heads you want, and pass them to load entity function.

And we'll have 3 ways to find heads:

1. Find draft-only head.
2. Find all non-draft heads.
3. Find only trusted heads.

WDYT @juligasa?